roles/kubernetes/preinstall/tasks/verify-settings.yml

```diff
@@ -17,13 +17,13 @@
 
 - name: Stop if unknown network plugin
   assert:
-    that: network_plugin in ['calico', 'canal', 'flannel', 'weave', 'cloud']
-  when: network_plugin is defined
+    that: kube_network_plugin in ['calico', 'canal', 'flannel', 'weave', 'cloud']
```

Comment:
cilium and contiv is missing. Please add :)

Reply:
done